README.md

```diff
@@ -1,6 +1,6 @@
 ## Open Referral Human Services Data API Specification
 
-This project is driven by an OpenAPI definition located in [_data/api-commons/openapi.yaml](https://github.com/openreferral/api-specification/blob/master/_data/api-commons/openapi.yaml)
+This project is driven by an OpenAPI definition located in [_data/api-commons/openapi.yaml](https://github.com/openreferral/api-specification/blob/master/_data/api-commons/openapi-hsda.yaml)
```

Comment:
@spilio I recommend that you change the link label in addition to the href.

Reply:
done.  thanks for pointing out!